<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ZIL1280" failures="1" tests="3" name="Guest User - Plans Availability-2" time="154.938" errors="0" timestamp="2023-12-18T00:57:54 IST">
  <testcase classname="com.perficient.util.TestCaseBase" name="@AfterMethod afterMethod" time="0.088">
    <failure type="java.lang.AssertionError" message="expected [pass] but found [fail]">
      <![CDATA[java.lang.AssertionError: expected [pass] but found [fail]
at com.perficient.util.TestCaseBase.afterMethod(TestCaseBase.java:205)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod afterMethod -->
  <testcase classname="com.perficient.testCases.Desktop.purdy.Verify_Login_Page_with_valid_credentials" name="customerLogin" time="19.772"/>
  <testcase classname="com.perficient.testCases.Desktop.purdy.Verify_Login_Page_with_valid_credentials" name="redirectToLoginPage" time="15.357"/>
  <testcase classname="com.perficient.testCases.Desktop.purdy.Verify_Login_Page_with_valid_credentials" name="homePageValidationAfterLogin" time="100.067">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at com.perficient.util.TestMethodListener.afterInvocation(TestMethodListener.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- homePageValidationAfterLogin -->
</testsuite> <!-- Guest User - Plans Availability-2 -->
